tests/playwright/src/model/ai-lab-start-recipe-page.ts

```diff
@@ -0,0 +1,55 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { expect as playExpect } from '@playwright/test';
+import type { Locator, Page } from '@playwright/test';
+import { AILabBasePage } from './ai-lab-base-page';
+import { handleConfirmationDialog } from '@podman-desktop/tests-playwright';
+
+export class AILabStartRecipePage extends AILabBasePage {
+  readonly recipeStatus: Locator;
+  readonly applicationDetailsPanel: Locator;
+  readonly startRecipeButton: Locator;
+  readonly openAIAppButton: Locator;
+  readonly deleteAIAppButton: Locator;
+
+  constructor(page: Page, webview: Page) {
+    super(page, webview);
+    this.heading = webview.getByLabel('Start Recipe');
```

Comment:
once you are using `this.` object to access webview, second time it is the passed parameter.

Reply:
good catch, thank you

Reply:
fixed